changelogs/fragments/5802-python-requirements-info-does-not-parse-extra-syntax.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+- python_requirements_info - fix bug with parsing project[extra] syntax
```

```suggestion
- python_requirements_info - fix bug with parsing ``project[extra]`` syntax (https://github.com/ansible-collections/community.general/pull/5802).
```
This PR is (or should be, if it fixes the bug) not just about parsing this syntax, but also returning information on such packages.